apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```diff
@@ -56,14 +56,18 @@
 	.template-selector-control__label {
 		display: block;
 		text-align: center;
-		border: 1px solid #999;
+		border: 1px solid transparent;
 		padding: 2em;
 		border-radius: 4px;
```

Comment:
All buttons in Gutenberg use pointer cursor. I'd add it here too to stay consistent.

```suggestion
		border-radius: 4px;
		cursor: pointer;
```